solana/bridge-escrow/programs/bridge-escrow/Cargo.toml

```diff
@@ -18,6 +18,7 @@ cpi = ["no-entrypoint"]
 default = []
 
 [dependencies]
+anyhow.workspace = true
```

Comment:
First of, this is only used in tests so should be in `[dev-dependencies]`. Second of, why not just `unwrap` in tests? We really don’t need error handling in a test.